package/peptidesim/utilities.py

```diff
@@ -39,3 +39,72 @@ def in_directory(file, directory, allow_symlink=False):
     # return true, if the common prefix of both is equal to directory
     # e.g. /a/b/c/d.rst and directory is /a/b, the common prefix is /a/b
     return os.path.commonprefix([file, directory]) == directory
+
+
+def validity_check(df, shifts):
+    Inval_index = []
+    Shif_index = []
+    count = 0
+    for i in range(len(shifts)):
+        Index = shifts[i]
+        Shif_index.append(Index+1)
+        if df.values[Index][1] == 1:
+            Inval_index.append(Index+1)
+            count += 1
+    return (count, Inval_index, Shif_index)
+
+
+def cs_validity(plumed_dat, exec_dir):
+    """plumed_dat = The path to the plumed file (include the plumed data filename and extension)
+       exec_dir = The path to the cs_shifts data files)"""
+    file_plumed = open(plumed_dat, 'r')
+    shifts_plumed = parser(file_plumed)
+    file_plumed.close()
+    keys = ['CAshifts', 'CBshifts', 'Cshifts',
+            'HAshifts', 'Hshifts', 'Nshifts']
+    shifts_plumed_dict = dict.fromkeys(keys)
+    dat_files = [f for f in os.listdir(exec_dir) if f.endswith('.dat')]
+    for i in range(len(shifts_plumed)):
+        shifts_dat = exec_dir + dat_files[i]
+        df_shifts_dat = pd.read_csv(shifts_dat, sep=" ", header=None)
+        shifts_plumed_dict[keys[i]] = shifts_plumed[i]
+        if shifts_plumed[i] != []:
+            count, Invalid_index, Shifts_index = validity_check(
+                df_shifts_dat, shifts_plumed_dict[keys[i]])
+            if count != 0:
+                raise ValueError('Simulation uses data {} for {}. Invalid EDS chemical shift found for data {}!'.format(
+                    Shifts_index, keys[i], Invalid_index))
+    print('All chemical shifts are valid'.)
```

Comment:
Can you just return a `True`? We don't use `print` in the peptidesim module, instead using a log file. The `print` statements end up in the slurm file.

Reply:
Also the period is outside the string, giving me a syntax error.